custom_components/afvalinfo/translations/en.json

```diff
@@ -23,13 +23,13 @@
 				"name": "Kerstboom"
 			},
 			"afvalinfo_papier": {
-				"name": "Papier"
+				"name": "Pape"
```

Comment:
Typo. "Pape" should be "Paper"

Reply:
Thanks for spotting, should be OK with https://github.com/heyajohnny/afvalinfo/pull/403/commits/eb559ee151a76e3d7ef1cb049e8968ec3f3aa0ab